docs/Configuration.md

```diff
@@ -384,6 +384,19 @@ Default: `false`
 
 Activates notifications for test results.
 
+### `notifyMode` [string]
+
+Default: `always`
+
+Specifies notification mode.
+
+#### Modes
+
+* `always`: always send a notification.
+* `failure`: send a notification when tests fail.
+* `success`: send a notification when tests pass.
+* `change`: send a notification when the status changed.
```

Comment:
It's missing the mode I would use, which is "on every failure, and change to success". I want a notification that my tests are still red on every run until they're green, and when it turns green, but not on every run when it's green. I'm not sure how to best express it, though. Travis has `onSuccess` and `onFailure` with different modes, but it feels a bit overkill to add two new options.

What about:  `always`, `failure`, `success`, `change`, `failure-always`, `failure-change`, `success-always`, `success-change`, and allow `notifyMode` to be passed an array instead of a single string?

I'm not sure if that's the best option, it's just what I could come up with without thinking too much about it

Reply:
What would be the difference between failure-always and failure (same with success and success-always)? I personally would prefer keeping it a string because I can only imagine 2 cases from the current combination of options. e.g. `['failure', 'change']` to accomplish the case you mentioned, and I think it would be better expressed as a string with `failure-change`

`success-change` would accomplish the inverse.

Reply:
>What would be the difference between failure-always and failure

Good point, those would be the same. `failure`, `success`, `always`, `change`, `failure-change` and `success-change` works for me

Reply:
Just added those :)